src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js

```diff
@@ -331,11 +333,14 @@ it('Verify how many work item type exists in drop down - desktop', function() {
                 expect(page.workItemTitle(page.firstWorkItem)).toBe(workItemTitle);
                 page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
                   var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
-                  detailPage.clickworkItemDetailTypeIcon();
-                  var wi_types = ["userstory", "valueproposition", "fundamental","experience","feature","bug"];
-                  for(var i=0;i<wi_types.length;i++){
-                   expect(detailPage.workItemTypeDropDownListString(wi_types[i]).getAttribute('innerHTML')).toContain(wi_types[i]);
-                  }
+                  //Commenting type drop down related tests for issue 635
+                  /*
+                    detailPage.clickworkItemDetailTypeIcon();
+                    var wi_types = ["userstory", "valueproposition", "fundamental","experience","feature","bug"];
+                    for(var i=0;i<wi_types.length;i++){
+                      expect(detailPage.workItemTypeDropDownListString(wi_types[i]).getAttribute('innerHTML')).toContain(wi_types[i]);
+                    }
+                  */
```

Comment:
I'd suggest just adding the `/* ... */` in two different lines (as you did) but leaving the actual code block's indentation as is, so that no DIFF is generated on that. It'll be easy again to restore them back the same way and with minimal DIFF.